frontend/src/components/App/Home/index.tsx

```diff
@@ -183,7 +183,7 @@ function HomeComponent(props: HomeComponentProps) {
   const { t } = useTranslation(['translation', 'glossary']);
   const [versions, errors] = useClustersVersion(Object.values(clusters));
   const maxWarnings = 50;
-  const warningsMap = Event.useWarningList(Object.values(customNameClusters).map(c => c.name));
+  const warningsMap = Event.useWarningListQuery(Object.values(customNameClusters).map(c => c.name));
```

Comment:
In this case we are adding the new query suffix. Why don't we keep the original name, as we did for the other functions?

Reply:
good point, I overlooked this. it's now using the original name